tests/e2e/elements/FormToggle.ts

```diff
@@ -0,0 +1,58 @@
+import { BaseElement } from './BaseElement';
+
+import { hasClass } from '../utils/actions';
+
+export class FormToggle extends BaseElement {
+	// Represents a FormToggle input. Use `selector` to represent the container its found in.
```

Comment:
The `switchOn` and `switchOff` buttons are quite nicely written, probably a bit safer then the `setCheckbox` & `unsetCheckbox` from the `@woocommerce/e2e-utils`, but it might still be worth to make use of those helper functions.

Reply:
@louwie17 If it's ok with you, I want to leave this and pick it up next time. I have spent a lot of time on this already and I'm keen to get it shipped for now. Especially because its such a large refactor that will impact other PRs.